### DISCUSSION TITLE
Connected and disconnected wav file added to A2DPsink

### DISCUSSION THREAD

**Discussion by joppes** [pschatzmann/arduino-audio-tools#983] [Q&A] [answered]
I'll like to add a signal to the I2S output when the A2DPsink is connected and a different sound when the BT link is disconnected. Can any anyone help me with this.

**Comment by pschatzmann** [comment, marked as answer]
I remember that was already answerd in the past. Did you search the discussions and issues in the A2DP project

https://github.com/pschatzmann/arduino-audio-tools#support

It would also help if you indicate where exactly you struggle: My first answer in a nutshell would be that this is very easy. Just play a sound (e.g. some specific frequency from the Sine Generator or a prerecorded sound from PROGMEM) dependent if you get a connect or disconnect event.
